### DISCUSSION TITLE
v1.0.0-beta.1

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#1063] [Announcements]
Major changes:

- Updates [shortcodes](https://v1--doks.netlify.app/docs/cheat-sheets/shortcodes/)
- Moves dependencies to [doks-core](https://github.com/gethyas/doks-core)
- Moves Doks configuration from [params.toml](https://github.com/h-enk/doks/blob/v1/config/_default/params.toml) to [doks.toml](https://github.com/h-enk/doks/blob/v1/config/_default/hyas/doks.toml)

<hr /><em>This discussion was created from the release <a href='https://github.com/h-enk/doks/releases/tag/v1.0.0-beta.1'>v1.0.0-beta.1</a>.</em>

**Comment by h-enk** [comment]
Preview at https://v1--doks.netlify.app/

**Comment by technosprout** [comment]
Please create a migration pathway from prior versions to the latest; many have existing codebases they have to migrate

**Reply by h-enk** [reply]
Will certainly do that /It's in the making — available soon!
